lib/main.js

```diff
@@ -17,4 +18,11 @@ const rss = require('./rss')
   const rssResult = rss(data)
   fs.writeFileSync(`${__dirname}/../out/feed_items.json`, rssResult.feed_items)
   fs.writeFileSync(`${__dirname}/../out/feed.xml`, rssResult.feed_xml)
+
+  const pre2017Result = await pre2017()
+  const pre2017JSON = pre2017Result.map(data => data.json)
+  fs.writeFileSync(
+    `${__dirname}/../out/pre2017.json`,
+    JSON.stringify(pre2017JSON)
```

Comment:
pretty print the JSON like the other emitted JSON files
(and the JSON should have an EOL at EOF)

Reply:
I can't find JSON.strigify option to add OEL